applications/StructuralMechanicsApplication/custom_advanced_constitutive/truss_plasticity_constitutive_law.cpp

```diff
@@ -207,9 +213,14 @@ array_1d<double, 3 > & TrussPlasticityConstitutiveLaw::CalculateValue(
     {
         if (rVariable == FORCE)
         {
+            const Properties& r_material_properties = rParameterValues.GetMaterialProperties();
+            double prestress = 0.00;
```

Comment:
since it is `r_material_properties` is `const` you can directly get it:
```suggestion
            const double prestress = r_material_properties[TRUSS_PRESTRESS_PK2];
```

Reply:
Ping